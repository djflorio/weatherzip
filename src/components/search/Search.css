.search {
  text-align: center;
}

.search__form {
  display: inline-block;
  max-width: 280px;
}

.search__country {
  margin-bottom: 10px;
}

.search__country-select {
  width: 100%;
  height: 40px;
  font-size: 1em;
  border-style: none;
  border-radius: 5px;
  background-color: #fff;
  box-shadow: 0 0 2px #222;
  animation: bounceIn 1s;
}

.search__zip {
  width: 100%;
  font-size: 1em;
  height: 40px;
  padding: 0 10px;
  box-sizing: border-box;
  border-style: none;
  border-radius: 5px;
  margin-bottom: 10px;
  box-shadow: 0 0 2px #222;
  animation: bounceIn 1s 0.2s both;
}

.search__submit {
  border-style: none;
  width: 100%;
  height: 40px;
  border-radius: 5px;
  font-size: 1em;
  background-color: #c5f8c5;
  cursor: pointer;
  transition: 0.3s;
  box-shadow: 0 0 2px #222;
  animation: bounceIn 1s 0.3s both;
}

.search__submit:hover {
  background-color: #92ff92;
}