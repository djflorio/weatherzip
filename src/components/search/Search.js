// Third party
import React from 'react';
import PropTypes from 'prop-types';

// Assets
import './Search.css';
import countries from '../../assets/countrycodes';


const UnitPicker = (props) => (
  <div className="unit-picker">
    <div
      className={"unit-picker__choice" + (props.units === "imperial" ? " unit-picker__choice--active" : "")}
      onClick={() => props.onChange("imperial")}>
      &deg;F
    </div>
    <div className={"unit-picker__choice" + (props.units === "metric" ? " unit-picker__choice--active" : "")}
      onClick={() => props.onChange("metric")}>
      &deg;C
    </div>
  </div>
);

const Search = (props) => {
  return (
    <div className="search">
      <form onSubmit={props.onSubmit} className="search__form">
        <div className="search__country">
          <select
            className="search__country-select"
            onChange={props.onChange}
            name="country"
            value={props.country}>
            {
              countries.map(country => (
                <option key={country[0]} value={country[0]}>{country[1]}</option>
              ))
            }
          </select>
        </div>
        <UnitPicker units={props.units} onChange={props.onUnitChange} />
        <input
          className="search__zip"
          name="zip"
          type="text"
          placeholder="Enter zip"
          value={props.zip}
          onChange={props.onChange}
        />
        <button type="submit" className="search__submit">
          Get Forecast
        </button>
      </form>
    </div>
  );
}

UnitPicker.propTypes = {
  units: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired
}

Search.propTypes = {
  country: PropTypes.string.isRequired,
  zip: PropTypes.string.isRequired,
  units: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired,
  onSubmit: PropTypes.func.isRequired,
  onUnitChange: PropTypes.func.isRequired
}

export default Search;