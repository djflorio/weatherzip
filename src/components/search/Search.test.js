import Search from './Search';

const initProps = {
  country: "US",
  zip: "10014",
  units: "imperial",
  onChange: () => {},
  onSubmit: () => {},
  onUnitChange: () => {}
};

function setupShallow(props = initProps) {
  return shallow(<Search {...props} />);
}

it("renders correctly", () => {
  const wrapper = setupShallow();
  expect(wrapper).toMatchSnapshot();
});

it("calls onChange when country changed", () => {
  const spy = sinon.spy();
  const props = {
    ...initProps,
    onChange: spy
  };
  const wrapper = setupShallow(props);

  wrapper
    .find(".search__country-select")
    .simulate("change");

  expect(spy.calledOnce).toBe(true);
});

it("calls onChange when zip changed", () => {
  const spy = sinon.spy();
  const props = {
    ...initProps,
    onChange: spy
  };
  const wrapper = setupShallow(props);

  wrapper
    .find(".search__zip")
    .simulate("change");

  expect(spy.calledOnce).toBe(true);
});

it("calls onSubmit when submitted", () => {
  const spy = sinon.spy();
  const props = {
    ...initProps,
    onSubmit: spy
  };
  const wrapper = setupShallow(props);

  wrapper
    .find(".search__form")
    .simulate("submit");

  expect(spy.calledOnce).toBe(true);
});