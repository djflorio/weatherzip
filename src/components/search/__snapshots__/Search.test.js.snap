// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="search"
>
  <form
    className="search__form"
    onSubmit={[Function]}
  >
    <div
      className="search__country"
    >
      <select
        className="search__country-select"
        name="country"
        onChange={[Function]}
        value="US"
      >
        <option
          key="AF"
          value="AF"
        >
          Afghanistan
        </option>
        <option
          key="AX"
          value="AX"
        >
          Aland Islands
        </option>
        <option
          key="AL"
          value="AL"
        >
          Albania
        </option>
        <option
          key="DZ"
          value="DZ"
        >
          Algeria
        </option>
        <option
          key="AS"
          value="AS"
        >
          American Samoa
        </option>
        <option
          key="AD"
          value="AD"
        >
          Andorra
        </option>
        <option
          key="AO"
          value="AO"
        >
          Angola
        </option>
        <option
          key="AI"
          value="AI"
        >
          Anguilla
        </option>
        <option
          key="AQ"
          value="AQ"
        >
          Antarctica
        </option>
        <option
          key="AG"
          value="AG"
        >
          Antigua And Barbuda
        </option>
        <option
          key="AR"
          value="AR"
        >
          Argentina
        </option>
        <option
          key="AM"
          value="AM"
        >
          Armenia
        </option>
        <option
          key="AW"
          value="AW"
        >
          Aruba
        </option>
        <option
          key="AU"
          value="AU"
        >
          Australia
        </option>
        <option
          key="AT"
          value="AT"
        >
          Austria
        </option>
        <option
          key="AZ"
          value="AZ"
        >
          Azerbaijan
        </option>
        <option
          key="BS"
          value="BS"
        >
          Bahamas
        </option>
        <option
          key="BH"
          value="BH"
        >
          Bahrain
        </option>
        <option
          key="BD"
          value="BD"
        >
          Bangladesh
        </option>
        <option
          key="BB"
          value="BB"
        >
          Barbados
        </option>
        <option
          key="BY"
          value="BY"
        >
          Belarus
        </option>
        <option
          key="BE"
          value="BE"
        >
          Belgium
        </option>
        <option
          key="BZ"
          value="BZ"
        >
          Belize
        </option>
        <option
          key="BJ"
          value="BJ"
        >
          Benin
        </option>
        <option
          key="BM"
          value="BM"
        >
          Bermuda
        </option>
        <option
          key="BT"
          value="BT"
        >
          Bhutan
        </option>
        <option
          key="BO"
          value="BO"
        >
          Bolivia
        </option>
        <option
          key="BA"
          value="BA"
        >
          Bosnia And Herzegovina
        </option>
        <option
          key="BW"
          value="BW"
        >
          Botswana
        </option>
        <option
          key="BV"
          value="BV"
        >
          Bouvet Island
        </option>
        <option
          key="BR"
          value="BR"
        >
          Brazil
        </option>
        <option
          key="IO"
          value="IO"
        >
          British Indian Ocean Territory
        </option>
        <option
          key="BN"
          value="BN"
        >
          Brunei Darussalam
        </option>
        <option
          key="BG"
          value="BG"
        >
          Bulgaria
        </option>
        <option
          key="BF"
          value="BF"
        >
          Burkina Faso
        </option>
        <option
          key="BI"
          value="BI"
        >
          Burundi
        </option>
        <option
          key="KH"
          value="KH"
        >
          Cambodia
        </option>
        <option
          key="CM"
          value="CM"
        >
          Cameroon
        </option>
        <option
          key="CA"
          value="CA"
        >
          Canada
        </option>
        <option
          key="CV"
          value="CV"
        >
          Cape Verde
        </option>
        <option
          key="KY"
          value="KY"
        >
          Cayman Islands
        </option>
        <option
          key="CF"
          value="CF"
        >
          Central African Republic
        </option>
        <option
          key="TD"
          value="TD"
        >
          Chad
        </option>
        <option
          key="CL"
          value="CL"
        >
          Chile
        </option>
        <option
          key="CN"
          value="CN"
        >
          China
        </option>
        <option
          key="CX"
          value="CX"
        >
          Christmas Island
        </option>
        <option
          key="CC"
          value="CC"
        >
          Cocos (Keeling) Islands
        </option>
        <option
          key="CO"
          value="CO"
        >
          Colombia
        </option>
        <option
          key="KM"
          value="KM"
        >
          Comoros
        </option>
        <option
          key="CG"
          value="CG"
        >
          Congo
        </option>
        <option
          key="CD"
          value="CD"
        >
          Congo, Democratic Republic
        </option>
        <option
          key="CK"
          value="CK"
        >
          Cook Islands
        </option>
        <option
          key="CR"
          value="CR"
        >
          Costa Rica
        </option>
        <option
          key="CI"
          value="CI"
        >
          Cote D"Ivoire
        </option>
        <option
          key="HR"
          value="HR"
        >
          Croatia
        </option>
        <option
          key="CU"
          value="CU"
        >
          Cuba
        </option>
        <option
          key="CY"
          value="CY"
        >
          Cyprus
        </option>
        <option
          key="CZ"
          value="CZ"
        >
          Czech Republic
        </option>
        <option
          key="DK"
          value="DK"
        >
          Denmark
        </option>
        <option
          key="DJ"
          value="DJ"
        >
          Djibouti
        </option>
        <option
          key="DM"
          value="DM"
        >
          Dominica
        </option>
        <option
          key="DO"
          value="DO"
        >
          Dominican Republic
        </option>
        <option
          key="EC"
          value="EC"
        >
          Ecuador
        </option>
        <option
          key="EG"
          value="EG"
        >
          Egypt
        </option>
        <option
          key="SV"
          value="SV"
        >
          El Salvador
        </option>
        <option
          key="GQ"
          value="GQ"
        >
          Equatorial Guinea
        </option>
        <option
          key="ER"
          value="ER"
        >
          Eritrea
        </option>
        <option
          key="EE"
          value="EE"
        >
          Estonia
        </option>
        <option
          key="ET"
          value="ET"
        >
          Ethiopia
        </option>
        <option
          key="FK"
          value="FK"
        >
          Falkland Islands (Malvinas)
        </option>
        <option
          key="FO"
          value="FO"
        >
          Faroe Islands
        </option>
        <option
          key="FJ"
          value="FJ"
        >
          Fiji
        </option>
        <option
          key="FI"
          value="FI"
        >
          Finland
        </option>
        <option
          key="FR"
          value="FR"
        >
          France
        </option>
        <option
          key="GF"
          value="GF"
        >
          French Guiana
        </option>
        <option
          key="PF"
          value="PF"
        >
          French Polynesia
        </option>
        <option
          key="TF"
          value="TF"
        >
          French Southern Territories
        </option>
        <option
          key="GA"
          value="GA"
        >
          Gabon
        </option>
        <option
          key="GM"
          value="GM"
        >
          Gambia
        </option>
        <option
          key="GE"
          value="GE"
        >
          Georgia
        </option>
        <option
          key="DE"
          value="DE"
        >
          Germany
        </option>
        <option
          key="GH"
          value="GH"
        >
          Ghana
        </option>
        <option
          key="GI"
          value="GI"
        >
          Gibraltar
        </option>
        <option
          key="GR"
          value="GR"
        >
          Greece
        </option>
        <option
          key="GL"
          value="GL"
        >
          Greenland
        </option>
        <option
          key="GD"
          value="GD"
        >
          Grenada
        </option>
        <option
          key="GP"
          value="GP"
        >
          Guadeloupe
        </option>
        <option
          key="GU"
          value="GU"
        >
          Guam
        </option>
        <option
          key="GT"
          value="GT"
        >
          Guatemala
        </option>
        <option
          key="GG"
          value="GG"
        >
          Guernsey
        </option>
        <option
          key="GN"
          value="GN"
        >
          Guinea
        </option>
        <option
          key="GW"
          value="GW"
        >
          Guinea-Bissau
        </option>
        <option
          key="GY"
          value="GY"
        >
          Guyana
        </option>
        <option
          key="HT"
          value="HT"
        >
          Haiti
        </option>
        <option
          key="HM"
          value="HM"
        >
          Heard Island & Mcdonald Islands
        </option>
        <option
          key="VA"
          value="VA"
        >
          Holy See (Vatican City State)
        </option>
        <option
          key="HN"
          value="HN"
        >
          Honduras
        </option>
        <option
          key="HK"
          value="HK"
        >
          Hong Kong
        </option>
        <option
          key="HU"
          value="HU"
        >
          Hungary
        </option>
        <option
          key="IS"
          value="IS"
        >
          Iceland
        </option>
        <option
          key="IN"
          value="IN"
        >
          India
        </option>
        <option
          key="ID"
          value="ID"
        >
          Indonesia
        </option>
        <option
          key="IR"
          value="IR"
        >
          Iran, Islamic Republic Of
        </option>
        <option
          key="IQ"
          value="IQ"
        >
          Iraq
        </option>
        <option
          key="IE"
          value="IE"
        >
          Ireland
        </option>
        <option
          key="IM"
          value="IM"
        >
          Isle Of Man
        </option>
        <option
          key="IL"
          value="IL"
        >
          Israel
        </option>
        <option
          key="IT"
          value="IT"
        >
          Italy
        </option>
        <option
          key="JM"
          value="JM"
        >
          Jamaica
        </option>
        <option
          key="JP"
          value="JP"
        >
          Japan
        </option>
        <option
          key="JE"
          value="JE"
        >
          Jersey
        </option>
        <option
          key="JO"
          value="JO"
        >
          Jordan
        </option>
        <option
          key="KZ"
          value="KZ"
        >
          Kazakhstan
        </option>
        <option
          key="KE"
          value="KE"
        >
          Kenya
        </option>
        <option
          key="KI"
          value="KI"
        >
          Kiribati
        </option>
        <option
          key="KR"
          value="KR"
        >
          Korea
        </option>
        <option
          key="KW"
          value="KW"
        >
          Kuwait
        </option>
        <option
          key="KG"
          value="KG"
        >
          Kyrgyzstan
        </option>
        <option
          key="LA"
          value="LA"
        >
          Lao People"s Democratic Republic
        </option>
        <option
          key="LV"
          value="LV"
        >
          Latvia
        </option>
        <option
          key="LB"
          value="LB"
        >
          Lebanon
        </option>
        <option
          key="LS"
          value="LS"
        >
          Lesotho
        </option>
        <option
          key="LR"
          value="LR"
        >
          Liberia
        </option>
        <option
          key="LY"
          value="LY"
        >
          Libyan Arab Jamahiriya
        </option>
        <option
          key="LI"
          value="LI"
        >
          Liechtenstein
        </option>
        <option
          key="LT"
          value="LT"
        >
          Lithuania
        </option>
        <option
          key="LU"
          value="LU"
        >
          Luxembourg
        </option>
        <option
          key="MO"
          value="MO"
        >
          Macao
        </option>
        <option
          key="MK"
          value="MK"
        >
          Macedonia
        </option>
        <option
          key="MG"
          value="MG"
        >
          Madagascar
        </option>
        <option
          key="MW"
          value="MW"
        >
          Malawi
        </option>
        <option
          key="MY"
          value="MY"
        >
          Malaysia
        </option>
        <option
          key="MV"
          value="MV"
        >
          Maldives
        </option>
        <option
          key="ML"
          value="ML"
        >
          Mali
        </option>
        <option
          key="MT"
          value="MT"
        >
          Malta
        </option>
        <option
          key="MH"
          value="MH"
        >
          Marshall Islands
        </option>
        <option
          key="MQ"
          value="MQ"
        >
          Martinique
        </option>
        <option
          key="MR"
          value="MR"
        >
          Mauritania
        </option>
        <option
          key="MU"
          value="MU"
        >
          Mauritius
        </option>
        <option
          key="YT"
          value="YT"
        >
          Mayotte
        </option>
        <option
          key="MX"
          value="MX"
        >
          Mexico
        </option>
        <option
          key="FM"
          value="FM"
        >
          Micronesia, Federated States Of
        </option>
        <option
          key="MD"
          value="MD"
        >
          Moldova
        </option>
        <option
          key="MC"
          value="MC"
        >
          Monaco
        </option>
        <option
          key="MN"
          value="MN"
        >
          Mongolia
        </option>
        <option
          key="ME"
          value="ME"
        >
          Montenegro
        </option>
        <option
          key="MS"
          value="MS"
        >
          Montserrat
        </option>
        <option
          key="MA"
          value="MA"
        >
          Morocco
        </option>
        <option
          key="MZ"
          value="MZ"
        >
          Mozambique
        </option>
        <option
          key="MM"
          value="MM"
        >
          Myanmar
        </option>
        <option
          key="NA"
          value="NA"
        >
          Namibia
        </option>
        <option
          key="NR"
          value="NR"
        >
          Nauru
        </option>
        <option
          key="NP"
          value="NP"
        >
          Nepal
        </option>
        <option
          key="NL"
          value="NL"
        >
          Netherlands
        </option>
        <option
          key="AN"
          value="AN"
        >
          Netherlands Antilles
        </option>
        <option
          key="NC"
          value="NC"
        >
          New Caledonia
        </option>
        <option
          key="NZ"
          value="NZ"
        >
          New Zealand
        </option>
        <option
          key="NI"
          value="NI"
        >
          Nicaragua
        </option>
        <option
          key="NE"
          value="NE"
        >
          Niger
        </option>
        <option
          key="NG"
          value="NG"
        >
          Nigeria
        </option>
        <option
          key="NU"
          value="NU"
        >
          Niue
        </option>
        <option
          key="NF"
          value="NF"
        >
          Norfolk Island
        </option>
        <option
          key="MP"
          value="MP"
        >
          Northern Mariana Islands
        </option>
        <option
          key="NO"
          value="NO"
        >
          Norway
        </option>
        <option
          key="OM"
          value="OM"
        >
          Oman
        </option>
        <option
          key="PK"
          value="PK"
        >
          Pakistan
        </option>
        <option
          key="PW"
          value="PW"
        >
          Palau
        </option>
        <option
          key="PS"
          value="PS"
        >
          Palestinian Territory, Occupied
        </option>
        <option
          key="PA"
          value="PA"
        >
          Panama
        </option>
        <option
          key="PG"
          value="PG"
        >
          Papua New Guinea
        </option>
        <option
          key="PY"
          value="PY"
        >
          Paraguay
        </option>
        <option
          key="PE"
          value="PE"
        >
          Peru
        </option>
        <option
          key="PH"
          value="PH"
        >
          Philippines
        </option>
        <option
          key="PN"
          value="PN"
        >
          Pitcairn
        </option>
        <option
          key="PL"
          value="PL"
        >
          Poland
        </option>
        <option
          key="PT"
          value="PT"
        >
          Portugal
        </option>
        <option
          key="PR"
          value="PR"
        >
          Puerto Rico
        </option>
        <option
          key="QA"
          value="QA"
        >
          Qatar
        </option>
        <option
          key="RE"
          value="RE"
        >
          Reunion
        </option>
        <option
          key="RO"
          value="RO"
        >
          Romania
        </option>
        <option
          key="RU"
          value="RU"
        >
          Russian Federation
        </option>
        <option
          key="RW"
          value="RW"
        >
          Rwanda
        </option>
        <option
          key="BL"
          value="BL"
        >
          Saint Barthelemy
        </option>
        <option
          key="SH"
          value="SH"
        >
          Saint Helena
        </option>
        <option
          key="KN"
          value="KN"
        >
          Saint Kitts And Nevis
        </option>
        <option
          key="LC"
          value="LC"
        >
          Saint Lucia
        </option>
        <option
          key="MF"
          value="MF"
        >
          Saint Martin
        </option>
        <option
          key="PM"
          value="PM"
        >
          Saint Pierre And Miquelon
        </option>
        <option
          key="VC"
          value="VC"
        >
          Saint Vincent And Grenadines
        </option>
        <option
          key="WS"
          value="WS"
        >
          Samoa
        </option>
        <option
          key="SM"
          value="SM"
        >
          San Marino
        </option>
        <option
          key="ST"
          value="ST"
        >
          Sao Tome And Principe
        </option>
        <option
          key="SA"
          value="SA"
        >
          Saudi Arabia
        </option>
        <option
          key="SN"
          value="SN"
        >
          Senegal
        </option>
        <option
          key="RS"
          value="RS"
        >
          Serbia
        </option>
        <option
          key="SC"
          value="SC"
        >
          Seychelles
        </option>
        <option
          key="SL"
          value="SL"
        >
          Sierra Leone
        </option>
        <option
          key="SG"
          value="SG"
        >
          Singapore
        </option>
        <option
          key="SK"
          value="SK"
        >
          Slovakia
        </option>
        <option
          key="SI"
          value="SI"
        >
          Slovenia
        </option>
        <option
          key="SB"
          value="SB"
        >
          Solomon Islands
        </option>
        <option
          key="SO"
          value="SO"
        >
          Somalia
        </option>
        <option
          key="ZA"
          value="ZA"
        >
          South Africa
        </option>
        <option
          key="GS"
          value="GS"
        >
          South Georgia And Sandwich Isl.
        </option>
        <option
          key="ES"
          value="ES"
        >
          Spain
        </option>
        <option
          key="LK"
          value="LK"
        >
          Sri Lanka
        </option>
        <option
          key="SD"
          value="SD"
        >
          Sudan
        </option>
        <option
          key="SR"
          value="SR"
        >
          Suriname
        </option>
        <option
          key="SJ"
          value="SJ"
        >
          Svalbard And Jan Mayen
        </option>
        <option
          key="SZ"
          value="SZ"
        >
          Swaziland
        </option>
        <option
          key="SE"
          value="SE"
        >
          Sweden
        </option>
        <option
          key="CH"
          value="CH"
        >
          Switzerland
        </option>
        <option
          key="SY"
          value="SY"
        >
          Syrian Arab Republic
        </option>
        <option
          key="TW"
          value="TW"
        >
          Taiwan
        </option>
        <option
          key="TJ"
          value="TJ"
        >
          Tajikistan
        </option>
        <option
          key="TZ"
          value="TZ"
        >
          Tanzania
        </option>
        <option
          key="TH"
          value="TH"
        >
          Thailand
        </option>
        <option
          key="TL"
          value="TL"
        >
          Timor-Leste
        </option>
        <option
          key="TG"
          value="TG"
        >
          Togo
        </option>
        <option
          key="TK"
          value="TK"
        >
          Tokelau
        </option>
        <option
          key="TO"
          value="TO"
        >
          Tonga
        </option>
        <option
          key="TT"
          value="TT"
        >
          Trinidad And Tobago
        </option>
        <option
          key="TN"
          value="TN"
        >
          Tunisia
        </option>
        <option
          key="TR"
          value="TR"
        >
          Turkey
        </option>
        <option
          key="TM"
          value="TM"
        >
          Turkmenistan
        </option>
        <option
          key="TC"
          value="TC"
        >
          Turks And Caicos Islands
        </option>
        <option
          key="TV"
          value="TV"
        >
          Tuvalu
        </option>
        <option
          key="UG"
          value="UG"
        >
          Uganda
        </option>
        <option
          key="UA"
          value="UA"
        >
          Ukraine
        </option>
        <option
          key="AE"
          value="AE"
        >
          United Arab Emirates
        </option>
        <option
          key="GB"
          value="GB"
        >
          United Kingdom
        </option>
        <option
          key="US"
          value="US"
        >
          United States
        </option>
        <option
          key="UM"
          value="UM"
        >
          United States Outlying Islands
        </option>
        <option
          key="UY"
          value="UY"
        >
          Uruguay
        </option>
        <option
          key="UZ"
          value="UZ"
        >
          Uzbekistan
        </option>
        <option
          key="VU"
          value="VU"
        >
          Vanuatu
        </option>
        <option
          key="VE"
          value="VE"
        >
          Venezuela
        </option>
        <option
          key="VN"
          value="VN"
        >
          Viet Nam
        </option>
        <option
          key="VG"
          value="VG"
        >
          Virgin Islands, British
        </option>
        <option
          key="VI"
          value="VI"
        >
          Virgin Islands, U.S.
        </option>
        <option
          key="WF"
          value="WF"
        >
          Wallis And Futuna
        </option>
        <option
          key="EH"
          value="EH"
        >
          Western Sahara
        </option>
        <option
          key="YE"
          value="YE"
        >
          Yemen
        </option>
        <option
          key="ZM"
          value="ZM"
        >
          Zambia
        </option>
        <option
          key="ZW"
          value="ZW"
        >
          Zimbabwe
        </option>
      </select>
    </div>
    <UnitPicker
      onChange={[Function]}
      units="imperial"
    />
    <input
      className="search__zip"
      name="zip"
      onChange={[Function]}
      placeholder="Enter zip"
      type="text"
      value="10014"
    />
    <button
      className="search__submit"
      type="submit"
    >
      Get Forecast
    </button>
  </form>
</div>
`;
